dask/dataframe/tests/test_groupby.py

```diff
@@ -2599,6 +2599,61 @@ def test_groupby_aggregate_categoricals(grouping, agg):
     assert_eq(agg(grouping(pdf)["value"]), agg(grouping(ddf)["value"]))
 
 
+@pytest.mark.parametrize(
+    "agg",
+    [
+        lambda grp: grp.agg(partial(np.std, ddof=1)),
+        lambda grp: grp.agg(partial(np.std, ddof=-2)),
+        lambda grp: grp.agg(partial(np.var, ddof=1)),
+        lambda grp: grp.agg(partial(np.var, ddof=-2)),
+    ],
+)
+def test_groupby_aggregate_partial_function(agg):
+    pdf = pd.DataFrame(
+        {
+            "a": [5, 4, 3, 5, 4, 2, 3, 2],
+            "b": [1, 2, 5, 6, 9, 2, 6, 8],
+        }
+    )
+    ddf = dd.from_pandas(pdf, npartitions=2)
+
+    # DataFrameGroupBy
+    assert_eq(agg(pdf.groupby("a")), agg(ddf.groupby("a")))
+
+    # SeriesGroupBy
+    assert_eq(agg(pdf.groupby("a")["b"]), agg(ddf.groupby("a")["b"]))
+
+
+@pytest.mark.parametrize(
+    "agg",
+    [
+        lambda grp: grp.agg(partial(np.std, unexpected_arg=1)),
+        lambda grp: grp.agg(partial(np.var, unexpected_arg=1)),
+    ],
+)
+def test_groupby_aggregate_partial_function_unexpected_args(agg):
+    pdf = pd.DataFrame(
+        {
+            "a": [5, 4, 3, 5, 4, 2, 3, 2],
+            "b": [1, 2, 5, 6, 9, 2, 6, 8],
+        }
+    )
+    ddf = dd.from_pandas(pdf, npartitions=2)
+
+    with pytest.raises((FutureWarning, TypeError)):
+        agg(pdf.groupby("a"))
+
+    with pytest.raises((FutureWarning, TypeError)):
```

Comment:
Couple of comments here:
- I might be missing something but I was not able to replicate a case where I saw the `FutureWarning` would you mind sharing why is this one here? 

- Can we include something like `match= "unexpected keyword argument"` to the `TypeError` like we do here for example https://github.com/dask/dask/blob/e57a0fb4026f79fb5bc6d2816d5351274dbfc7a4/dask/dataframe/tests/test_dataframe.py#L812-L813

Reply:
Test cases are updated.

The `FutureWarning` was odd. It showed up in the unit test, but now I can't reproduce it on a simple example either. I removed it from the test.